src/virtual-select.js

```diff
@@ -258,7 +258,7 @@ export class VirtualSelect {
     const { convertToBoolean } = Utils;
 
     if (markSearchResults) {
-      searchRegex = new RegExp(`(${Utils.regexEscape(this.searchValue)})`, 'gi');
+      searchRegex = new RegExp(`(${Utils.regexEscape(this.searchValue)})(?!([^<]+)?>)`, 'gi');
```

Comment:
For performance, you could store the regex on a const

Reply:
Three comments:
- Why was this change done on this method, instead of directly in the util **regexEscape** given the fact that it's only used on this place?
- I would still say to store the regex on a const so that the jit doesn't need to define a variable on each iteration. 
- could you add some test scenarios to the PR to show what is this solving and how.

Reply:
In order to speed up the release I'll merge this and review after merge